templates/coredns/0/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+coredns:
+  image: coredns/coredns:1.0.1
+  labels:
+    io.rancher.sidekicks: data
+    io.rancher.container.hostname_override: container_name
+  command: [ "-conf", "/etc/coredns/Corefile" ]
+  ports:
+    - 53:53/tcp
+    - 53:53/udp
+  volumes_from:
+    - data
+data:
+  image: zhenyangzhao/coredns-file:v0.8.0
```

Comment:
I am not sure if you need a different image here.
Can you reuse the `coredns/coredns` image? If not, it's ok.
It's up to you.

Reply:
Sorry this can not reuse.